bash_completions.go

```diff
@@ -495,12 +495,15 @@ func writeFlag(buf *bytes.Buffer, flag *pflag.Flag, cmd *Command) {
 
 func writeLocalNonPersistentFlag(buf *bytes.Buffer, flag *pflag.Flag) {
 	name := flag.Name
-	format := "    local_nonpersistent_flags+=(\"--%s"
+	format := "    local_nonpersistent_flags+=(\"--%[1]s"
 	if len(flag.NoOptDefVal) == 0 {
-		format += "="
+		format += "\")\n    local_nonpersistent_flags+=(\"--%[1]s="
 	}
 	format += "\")\n"
 	buf.WriteString(fmt.Sprintf(format, name))
```

Comment:
This works nicely, but I think we can make it more readable with:
```
        name := flag.Name
	format := "    local_nonpersistent_flags+=(\"--%[1]s\")\n"
	if len(flag.NoOptDefVal) == 0 {
		format += "    local_nonpersistent_flags+=(\"--%[1]s=\")\n"
	}
	buf.WriteString(fmt.Sprintf(format, name))
```

What do you think?

Reply:
Yes sure.